packages/sanity/src/core/form/components/EditPortal.tsx

```diff
@@ -35,26 +36,32 @@ export function EditPortal(props: Props): React.ReactElement {
 
   if (type === 'dialog') {
     return (
-      <Dialog
+      <BoundaryElementProvider element={documentScrollElement.current}>
```

Comment:
should this be in state rather than a mutable ref? As it is now we won't trigger a new re-render if it changes, right?

Reply:
🤔 hmm not sure to be honest it seems to be working but I'll still change to keep it consistent with DocumentPanel implementation and possibly what you mentioned